test/test_autograd.py

```diff
@@ -8616,7 +8616,8 @@ def test_copy_(self, device):
         # At the time of writing this test, copy_ is not generated from native_functions.yaml
         # there was a bug that bfloat16 was not recognized as floating.
         x = torch.randn(10, device=device, requires_grad=True)
-        floating_dt = [dt for dt in get_all_dtypes() if dt.is_floating_point]
+        floating_dt = [dt for dt in all_types_and_complex_and(torch.half, torch.bfloat16, torch.bool)
```

Comment:
This can just enumerate the the floating point dtypes directly

Reply:
Done!